src/confighelper.cpp

```diff
@@ -100,11 +100,20 @@ void ConfigHelper::importGuiConfigJson(const QString &file)
         } else {
             p.name = json["profile"].toString();
             p.serverPort = json["server_port"].toString().toUShort();
-            p.localAddress = json["local_address"].toString();
-            p.localPort = json["local_port"].toString().toUShort();
-            p.timeout = json["timeout"].toString().toInt();
+            p.localAddress = QString("127.0.0.1");
+            if (json.contains("local_address")) {
+                p.localAddress = json["local_address"].toString();
+            }
+            p.localPort = json["local_port"].toInt();
```

Comment:
`p.localPort` should be parsed as String then convert to unsigned short. `else` block applies to older `ss-qt5` config